luacode/formatter.lua

```diff
@@ -515,8 +517,11 @@ function M.indentcode(source, delimiter,indenttable, ...)
     end
 
     -- Check code validity
-    local status, message = loadstring(source,"isCodeValid")
-    if not status then return status, message end
+    local message = validate(source);
+    if message then return nil, message end
+
+    -- Special escape for '|' character
+    source = source:gsub('|', '/|')
```

Comment:
This allows me to define `/|` as an operator in the lexer which avoids the conflict with `|` used by MetaLua.

Reply:
On the off chance that this matches something in the source, do we want to use an even screwier character sequence? Maybe there's a non-printed character we could use?